superset-frontend/src/dashboard/components/Header/HeaderActionsDropdown/index.jsx

```diff
@@ -158,7 +158,7 @@ class HeaderActionsDropdown extends React.PureComponent {
     switch (key) {
       case MENU_KEYS.REFRESH_DASHBOARD:
         this.props.forceRefreshAllCharts();
-        this.props.addSuccessToast(t('Data refreshed'));
+        this.props.addSuccessToast(t('Refreshing Charts'));
```

Comment:
```suggestion
        this.props.addSuccessToast(t('Refreshing charts'));
```
We keep things sentence case unless the noun is referring to the page, e.g. the Charts page.